client/landing/stepper/declarative-flow/internals/steps-repository/unified-plans/index.tsx

```diff
@@ -58,7 +58,10 @@ function getPlansIntent( flowName: string | null, isWordCampPromo?: boolean ): P
 	}
 }
 
-export default function PlansStepAdaptor( props: StepProps ) {
+const PlansStepAdaptor: Step< {
+	// TODO: work on more specific types
```

Comment:
Maybe we can open a task instead of leaving a TODO comment

Reply:
https://github.com/Automattic/wp-calypso/issues/101497